webviews/src/views/RokuAutomationView/AutoRunsEditor.svelte

```diff
@@ -0,0 +1,395 @@
+<script>
+    import { Trash, Copy, TriangleDown } from 'svelte-codicons';
+    import { dropzone, draggable } from './dnd';
+
+    export let runs;
+    export let selectedRun;
+
+    let runTable;
+    let runNameInput;
+    let runNameDialog;
+    let runNameDialogValue;
+    let confirmDialog;
+    let showContent = false;
```

Comment:
Let's ensure that all variables in this file have a typescript type assigned.

Reply:
yep